extensions/banner/deployment/src/main/java/io/quarkus/banner/deployment/BannerProcessor.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.banner.deployment;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+import io.quarkus.banner.runtime.BannerRecorder;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.HotDeploymentWatchedFileBuildItem;
+import io.quarkus.deployment.util.FileUtil;
+
+public class BannerProcessor {
+
+    private static final String DEFAULT_BANNER_PATH = "src/main/resources/banner.txt";
```

Comment:
Probably make this the `defaultValue` for `BannerConfig #path` field so that it is properly  and automatically documented with our config doc generation infrastructure.

Reply:
Sure ! That makes more sense